test/com/dotcms/util/marshal/MarshalUtilsTest.java

```diff
@@ -35,6 +38,12 @@
  */
 
 public class MarshalUtilsTest {
+	
+	@BeforeClass
+	public static void prepare() throws Exception{
+		//Setting web app environment
+        IntegrationTestInitService.getInstance().init();
```

Comment:
needs to be fixed as well, @erickgonzalez can you please create issues for this one and JsonWebTokenInterceptorTest. They are true unit and nest to run ok again. Short description just say they are failing